lib/ModernTribe/ruleset.xml

```diff
@@ -59,6 +59,13 @@
        functions, methods, if, elseif, foreach, for, and switch blocks. -->
     <rule ref="WordPress.WhiteSpace.ControlStructureSpacing"/>
 
+    <!-- No short echos allowed. WordPress-VIP-GO overides the default 
+        severity.  -->
+    <rule ref="Generic.PHP.DisallowShortOpenTag.EchoFound">
```

Comment:
Actually, should move this to the proper `Generic` section below.

Reply:
Thanks, should be good now.